angular.json

```diff
@@ -25,13 +18,12 @@
             "main": "src/main.ts",
             "polyfills": "src/polyfills.ts",
             "tsConfig": "tsconfig.app.json",
-            "inlineStyleLanguage": "scss",
             "assets": [
               "src/favicon.ico",
               "src/assets"
             ],
             "styles": [
-              "src/styles.scss"
+              "src/styles.css"
```

Comment:
This and the other changes to this file looks like things are being changed from scss to css. I have a feeling most other teams prefer scss, so I think these changes need to be removed

Reply:
done